crates/rpc/rpc/src/debug.rs

```diff
@@ -659,6 +667,9 @@ where
                 let witness = state_provider
                     .witness(HashedPostState::default(), hashed_state)
                     .map_err(Into::into)?;
+
+                // TODO(alexey): return `state_preimages` as well when https://github.com/alloy-rs/alloy/pull/1178 is released
```

Comment:
let's maybe wait for the release? otherwise, it's collecting preimages for no reason

Reply:
sg